test/modules/Modules.Test/TestResultCoordinator/TestReportUtilTest.cs

```diff
@@ -308,16 +352,41 @@ public void ParseReportMetadataList_ParseDirectMethodTestReportMetadataWithoutRe
             List<ITestReportMetadata> results = TestReportUtil.ParseReportMetadataJson(testDataJson, new Mock<ILogger>().Object);
 
             Assert.Single(results);
-            var reportMetadata = results[0] as DirectMethodReportMetadata;
+            var reportMetadata = results[0] as DirectMethodConnectivityReportMetadata;
             Assert.NotNull(reportMetadata);
             Assert.Equal("edge agent ping", reportMetadata.TestDescription);
             Assert.Equal(TestOperationResultType.DirectMethod, reportMetadata.TestOperationResultType);
-            Assert.Equal(TestReportType.DirectMethodReport, reportMetadata.TestReportType);
+            Assert.Equal(TestReportType.DirectMethodConnectivityReport, reportMetadata.TestReportType);
             Assert.Equal("directMethodSender1.send", reportMetadata.SenderSource);
             Assert.False(reportMetadata.ReceiverSource.HasValue);
             Assert.Equal(new TimeSpan(0, 0, 0, 0, 5), reportMetadata.TolerancePeriod);
         }
 
+        [Fact]
+        public void ParseReportMetadataList_ParseDirectMethodLongHaulTestReportMetadataWithoutReceiverSource()
```

Comment:
Can you explain the context around these tests? In what scenario is a report made without the receiver? Is it when the direct method is initially sent?

Reply:
EdgeAgent ping tests is the only one I know with no receiver